src/ReactiveUI/Xaml/ViewModelViewHost.cs

```diff
@@ -101,21 +101,20 @@ public ViewModelViewHost()
                     }
 
                     var viewLocator = ViewLocator ?? ReactiveUI.ViewLocator.Current;
-                    var view = viewLocator.ResolveView(x.ViewModel, x.Contract) ?? viewLocator.ResolveView(x.ViewModel, null);
+                    var view = viewLocator.ResolveView(x.ViewModel, x.Contract) ?? viewLocator.ResolveView(x.ViewModel);
```

Comment:
@ghuntley the second parameter is optional and therefore always null, so it could be safely removed. It was my R# alt + return muscle memory that forces me to "fix" it ;-) same as with "this" and the string.Format, but if you prefer to keep it I'm fine with it too.